hilti/runtime/src/util.cc

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) 2020-2021 by the Zeek Project. See LICENSE for details.
 
+#include "util.h"
```

Comment:
```suggestion
#include <hilti/rt/util.h>
```

For consistency with other files.